src/components/charts/LoadingOverlay.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="loading-overlay" :style="style">
+    <v-icon size="x-large">mdi-loading mdi-spin</v-icon>
```

Comment:
There is also `<v-progress-circular indeterminate />` that we already use as a loading icon for the WMS layers, and which has a slightly cooler animation 😄

Reply:
Discussed with Tom. Use skeleton loader.